cfme/tests/services/test_rest_services.py

```diff
@@ -2402,3 +2402,35 @@ def test_edit_service_request_task(appliance, request_task):
     request_task.reload()
     assert request_task.options["request_param_a"] == "value_a"
     assert request_task.options["request_param_b"] == "value_b"
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1716847])
+@pytest.mark.customer_scenario
+def test_edit_ansible_playbook_cat_item_rest(appliance, request, ansible_catalog_item):
+    """
+    Bugzilla:
+        1716847
```

Comment:
**Suggestion:**
Please add Cloned BZ --> https://bugzilla.redhat.com/show_bug.cgi?id=1732117.

Reply:
@valaparthvi You might missed this :smile:

Reply:
I did not. I added it to the Bugzilla block. Since it's a cloned BZ, I did not add it to the `automates` block.

Reply:
@valaparthvi I am not suggesting to add into `automates` block, add it into `Bugzilla` block.

Reply:
I added it @sbulage, check the file.